test/e2e/servers/ory/kratos.spec.ts

```diff
@@ -517,3 +517,40 @@ describe("decoding link header", () => {
     expect(getNextPage(withoutNext)).toBe(undefined)
   })
 })
+
+describe("cookie flow", () => {
+  const authService = AuthWithPhonePasswordlessService()
+  const phone = randomPhone()
+  let cookies: Array<KratosCookie>
+  let cookieStr = ""
+
+  it("login with cookie", async () => {
+    const res = await authService.createIdentityWithCookie(phone)
+    if (res instanceof Error) throw res
+    expect(res).toHaveProperty("kratosUserId")
+    expect(res).toHaveProperty("cookiesToSendBackToClient")
+    cookies = res.cookiesToSendBackToClient
+  })
+
+  it("logout and revoke cookies", async () => {
+    for (const c of cookies) {
```

Comment:
I know it's just used for 2 lines, but AFAIK, we try to not use 1 letter variable in the codebase. 

`for (const cookie of cookies) {` would probably still fit in on a single line.